docs/nodes/instantiate-testnet.md

```diff
@@ -76,7 +76,7 @@ CELES_AMOUNT="10000000000000000000000000utia"
 celestia-appd add-genesis-account $KEY_NAME $CELES_AMOUNT --keyring-backend test
 ```
 
-Here `$VALIDATOR_NAME` is the same key name as before; and `$AMOUNT`
+Here `$KEY_NAME` is the same key name as before; and `$AMOUNT`
 is something like `10000000000000000000000000utia`.
```

Comment:
[nit][not blocking] the $AMOUNT flag appears to reference $CELES_AMOUNT and that's already re-defined in the code above so it doesn't appear to need extra explanation.

```suggestion
Here `$KEY_NAME` is the same key name as before.
```

Reply:
This looks correct based on https://github.com/celestiaorg/celestia-app/blob/e2b154d8314872eba6f0d8d5ea18fdb58b201150/scripts/single-node.sh#L16-L17

where $KEY_NAME is defined as validator above. It may make sense to set all the environment variables at the top of this file but just a thought